doc/en/user/source/extensions/css/workshop/polygon.rst

```diff
@@ -0,0 +1,880 @@
+      }
+
+#. The :guilabel:`Map` tab provides the same preview.
+
+   .. image:: img/polygon_10_recode_theme.png
+
+#. The :guilabel:`Generated SLD` tab shows where things get interesting. Our generated style now consists of a single **Rule**:
+
+   .. code-block:: xml
+
+      <sld:Rule>
+         <sld:PolygonSymbolizer>
+            <sld:Fill>
+               <sld:CssParameter name="fill">
+                  <ogc:Function name="Recode">
+                     <ogc:PropertyName>mapcolor9</ogc:PropertyName>
+                     <ogc:Literal>1</ogc:Literal>
+                        <ogc:Literal>#8dd3c7</ogc:Literal>
+                     <ogc:Literal>2</ogc:Literal>
+                        <ogc:Literal>#ffffb3</ogc:Literal>
+                     <ogc:Literal>3</ogc:Literal>
+                        <ogc:Literal>#bebada</ogc:Literal>
+                     <ogc:Literal>4</ogc:Literal>
+                        <ogc:Literal>#fb8072</ogc:Literal>
+                     <ogc:Literal>5</ogc:Literal>
+                        <ogc:Literal>#80b1d3</ogc:Literal>
+                     <ogc:Literal>6</ogc:Literal>
+                        <ogc:Literal>#fdb462</ogc:Literal>
+                     <ogc:Literal>7</ogc:Literal>
+                        <ogc:Literal>#b3de69</ogc:Literal>
+                     <ogc:Literal>8</ogc:Literal>
+                        <ogc:Literal>#fccde5</ogc:Literal>
+                     <ogc:Literal>9</ogc:Literal>
+                        <ogc:Literal>#d9d9d9</ogc:Literal>
+               </ogc:Function>
+               </sld:CssParameter>
+            </sld:Fill>
+         </sld:PolygonSymbolizer>
+         <sld:LineSymbolizer>
+            <sld:Stroke>
+               <sld:CssParameter name="stroke">#808080</sld:CssParameter>
+               <sld:CssParameter name="stroke-width">0.5</sld:CssParameter>
+            </sld:Stroke>
+         </sld:LineSymbolizer>
+      </sld:Rule>
+
+
+Additional Considerations
+-------------------------
+
+#. When we rendered our initial preview, without a stroke, thin white gaps (or slivers) are visible between our polygons.
+
+   .. image:: img/polygon_04_preview.png
+
+   This effect is made more pronounced by the rendering engine making use of the Java 2D sub-pixel accuracy. This technique is primarily used to prevent an aliased (stair-stepped) appearance on diagonal lines.
+
+#. Clients can turn this feature off using a GetMap format option::
+
+     format_options=antialiasing=off;
+
+   The **LayerPreview** provides access to this setting from the Open Layers **Options Toolbar**:
+
+   .. image:: img/polygon_antialias.png
+
+   To eliminate the slivers between polygons, we can use the following css:
+
+   .. code-block:: css
+
+    * {
+      fill: lightgrey;
+      stroke: lightgrey;
+    }
+
+#. The **Categorize** function can be used to generate property values based on quantitative information. Here is an example using Categorize to color states according to size.
```

Comment:
Hmmm, yeah good point. I've further subdivided it to hopefully make it flow better.